conditional/util/member.py

```diff
@@ -15,21 +16,32 @@
 from conditional.models.models import MemberCommitteeAttendance
 from conditional.models.models import TechnicalSeminar
 from conditional.models.models import HouseMeeting
+from conditional.models.models import CurrentCoops
 
 from conditional import start_of_year
 
 
 @lru_cache(maxsize=1024)
 def get_voting_members():
+
+    if datetime.today() < datetime(start_of_year().year, 12, 31):
+        semester = 'Fall'
+    else:
+        semester = 'Spring'
+
     voting_list = [uid for uid in [member.uid for member in ldap_get_active_members()]
-                   if uid not in [member.uid for member in ldap_get_intro_members()]]
+                   if uid not in [member.uid for member in ldap_get_intro_members()] and
+                   uid not in [member.uid for member in CurrentCoops.query.filter(
```

Comment:
I think this would be a good place for us to use a filter and a lambda function. 

Something where we get the relevant lists before and then just do a quick comparator seems more elegant.